packages/span-transformations/package.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "opentelemetry-span-transformations",
+    "version": "0.0.1",
+    "description": "Convert spans between different formats",
+    "author": "Aspecto (aspecto.io)",
+    "homepage": "www.aspecto.io",
+    "license": "Apache-2.0",
+    "main": "dist/src/index.js",
+    "directories": {
+        "lib": "src",
+        "test": "test"
+    },
```

Comment:
non of the other packages in this repo is using the feature.
What is the motivation?

Reply:
Well, when I created the package it was done using lerna create.
Apparently this info is not used by anything, but is a good metadata on our packages.
https://docs.npmjs.com/cli/v6/configuring-npm/package-json#directories

I could remove it if you for consistency with other packages, but other than that don't think it matters either way. 

what's your preference ? keep or remove?

Reply:
I think it's best to have the same behavior as other packages in the repo.
If you think it's useful, we can add it to all packages.

Reply:
Removed for consistency. i don't think it matters.